<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="authorEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>581</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAuthorButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>581</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить автора</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
